.storybook/components/Recipes/Recipes.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { StoryObj } from '@storybook/react';
+import React from 'react';
+import { RecipesDocumentation } from './RecipesDocumentation';
+import RecipesDocs from '../../../docs/RECIPES.md';
+
+export default {
+  title: '',
```

Comment:
Without a title, it automatically gets dumped in the "Recipes" folder by default.